BankManagerConsoleApp/BankManagerConsoleApp/Tellers.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+struct Tellers {
+    private let tellerCount: Int
+    private let tellerType: TypeOfWork
+    
+    init(tellerCount: Int, tellerType: TypeOfWork) {
+        self.tellerCount = tellerCount
+        self.tellerType = tellerType
+    }
+
+    func doTask(queue: Queue<Int>) {
+        let name = tellerType.name
+        let time = tellerType.time
+        
+        let semaphore = DispatchSemaphore(value: tellerCount)
+        let group = DispatchGroup()
+        var workingTellerCount = 0
+        
+        while !queue.isEmpty() {
+            semaphore.wait()
+            guard let data = queue.dequeue() else {
+                semaphore.signal()
+                return
+            }
```

Comment:
이 부분도 semaphore를 wait하지 않으면 signal 예외 처리를 할 필요가 없기 때문에, 다음과 같이 바꾸면 예외처리를 좀 더 줄일 수 있어요. 참고만하세요!

```suggestion
             guard let data = queue.dequeue() else {
                 return
             }
            semaphore.wait()

Reply:
예전처럼 dequeue할 때 비동기 처리를 안하기 때문에 semaphore.wait의 위치를 guard문 밑으로 옮겨서 불필요한 예외처리가 필요 없어 보이네요. 
감사합니다.